__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+
+afterAll(() =>
+{
+    db.end();
+})
+
+describe('topics', () =>
+{
+    describe.only('GET /api/topics', () =>
+    {
+        test('200: responds with array of topics', () =>
+        {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body: topics}) =>
+            {
+                expect(topics).toBeInstanceOf(Array)
```

Comment:
If this returned an empty array, it would still pass the forEach assertion below -> because at no point would it find an example of something that didn't include the required keys (because it just wouldn't find anything - but that in itself wouldn't cause an error!). In order to avoid false positives, please also test explicitly that the length of the array reflects what's in the database.